Patch by Robert Scheck <robert@fedoraproject.org> for gnokii <= 0.6.31 to avoid
build failures with GCC 5.x as "gnokii-0.6.31/xgnokii/xgnokii.c:1209: undefined
reference to `GUI_HideAbout'" and "xgnokii-xgnokii_logos.o:(.data.rel+0xe0):
undefined reference to `CloseLogosWindow'".

--- gnokii-0.6.31/xgnokii/xgnokii.c		2011-12-02 15:53:04.000000000 +0100
+++ gnokii-0.6.31/xgnokii/xgnokii.c.gcc5	2015-06-20 15:49:59.000000000 +0200
@@ -798,7 +798,7 @@
 }
 
 
-inline void GUI_HideAbout(void)
+void GUI_HideAbout(void)
 {
 	gtk_widget_hide(AboutDialog);
 }
--- gnokii-0.6.31/xgnokii/xgnokii_logos.c	2011-12-02 15:53:04.000000000 +0100
+++ gnokii-0.6.31/xgnokii/xgnokii_logos.c.gcc5	2015-06-20 15:50:15.000000000 +0200
@@ -1304,7 +1304,7 @@
 	return 0;
 }
 
-inline void CloseLogosWindow(void)
+void CloseLogosWindow(void)
 {
 	gtk_widget_hide(GUI_LogosWindow);
 }
